keel-web/src/main/kotlin/com/netflix/spinnaker/keel/rest/DeliveryConfigController.kt

```diff
@@ -34,6 +35,10 @@ class DeliveryConfigController(
   private val artifactRepository: ArtifactRepository,
   private val adHocDiffer: AdHocDiffer
 ) {
+  @Operation(
```

Comment:
Muuuuch better than that YAML file... ❤️

Reply:
Yeah, since the controllers will not be part of `keel-api` there’s no issue using annotations.